src/main/resources/react/src/components/header/nav_main/styles.js

```diff
@@ -0,0 +1,169 @@
+import styled from 'styled-components';
+import logo from 'assets/img/logo.png';
+import { Link } from 'react-router-dom';
+
+export const DivWrap = styled.div`
+  font-family: Roboto, sans-serif;
+  padding: 20px 0 22px;
+  border-radius: 0 0 20px 20px;
+  background-color: #fff;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.08);
+`;
+
+export const Wrapper = styled.div`
+  margin: 0 auto;
+  max-width: 1251px;
+  overflow: hidden;
+  padding: 0 15px;
+  width: 100%;
+`;
+
+export const WrapMain = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const LogoLink = styled(Link)`
+  display: block;
+  min-width: 148px;
+  width: 148px;
+  height: 56px;
+  background: url(${logo});
+  margin-right: 40px;
+
+  &:active {
+    transform: scale(1.03);
+  }
+  @media (min-width: 320px) {
+    display: none;
+  }
+  @media (min-width: 540px) {
+    display: block;
+  }
+`;
+
+export const WrapCategories = styled.div`
+  position: absolute;
+  height: 56px;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 24px;
+  color: ${({ theme: { colors } }) => colors.blackColorText};
+  margin: 25px 0 0 179px;
+  padding-bottom: 95px;
+  cursor: pointer;
+
+  &:hover {
+    left: 0;
+    transform: translateY(0);
+  }
+  @media (min-width: 320px) {
+    display: none;
+  }
+  @media (min-width: 540px) {
+    display: block;
+    margin-left: 149px;
+  }
+  @media (min-width: 840px) {
+    margin-left: 164px;
+  }
+  @media (min-width: 1200px) {
+    margin-left: 179px;
+  }
+`;
+
+export const WrapCategoriesTop = styled.div`
+  opacity: 0;
+  transform: translateY(-50%);
+  position: relative;
+  bottom: -13px;
+  color: ${({ theme: { colors } }) => colors.blackColorText};
+
+  &:after {
+    position: absolute;
+    right: -18px;
+    bottom: 9px;
+    content: '';
+    border: 5px solid transparent;
+    border-bottom: 5px solid ${({ theme: { colors } }) => colors.btnBlueActive};
+  }
```

Comment:
if in component exist 2 callbacks and more
we need 

${({ theme }) => css`
color: ${({ theme }) => theme.colors.blackColorText};

 &:after {
    position: absolute;
    right: -18px;
    bottom: 9px;
    content: '';
    border: 5px solid transparent;
    border-bottom: 5px solid ${({ theme }) => theme.colors.btnBlueActive};
  }
`}